aws.go

```diff
@@ -183,14 +197,18 @@ func scrape(collector *cwCollector, ch chan<- prometheus.Metric) {
 
 //Send a single dataPoint to the Prometheus lib
 func scrapeSingleDataPoint(collector *cwCollector, ch chan<- prometheus.Metric,params *cloudwatch.GetMetricStatisticsInput,metric *cwMetric,labels []string,svc *cloudwatch.CloudWatch) error {
+	now := time.Now()
 	resp, err := svc.GetMetricStatistics(params)
 	totalRequests.Inc()
-
+	collector.ScrapeDurationHistogram.Observe(time.Since(now).Seconds())
+	
 	if err != nil {
+		totalErrors.Inc()
 		collector.ErroneousRequests.Inc()
 		fmt.Println(err)
 		return err
 	}
+	collector.SuccessfulRequests.Inc()
```

Comment:
increment in 2s for err

Reply:
`return err` on error means func exits before L211 on err.